editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -135,7 +137,7 @@ class SnapDialog : public ConfirmationDialog {
 		label->set_h_size_flags(Control::SIZE_EXPAND_FILL);
 
 		grid_step_x = memnew(SpinBox);
-		grid_step_x->set_min(0.01);
+		grid_step_x->set_min(0);
```

Comment:
Wouldn't it make sense to set this minimum to 1 as well?

Reply:
Agree, I'd just move `SPIN_BOX_STEP_MIN` to the constructor (and rename it to `SPIN_BOX_GRID_STEP_MIN`) and use it only in the both `set_min` calls. Then the changes in `get_fields` shouldn't be needed anymore, could be reverted.

Reply:
> Wouldn't it make sense to set this minimum to 1 as well?

Yes, I did not think of that because the minimum value of the range was 0.01.
Thanks 👍, it's fixed now.